src/features/events/l10n/messageIds.ts

```diff
@@ -282,8 +282,9 @@ export default makeMessages('feat.events', {
   tooltipContent: m('Untitled events will display type as title'),
   type: {
     createType: m<{ type: string }>('Create "{type}"'),
-    deleteMessage: m<{ eventType: string }>(
-      'Are you sure you want to delete the "{eventType}" event type for the whole organization?'
+    currentDefaultOrgMsg: m('the current organization'),
+    deleteMessage: m<{ eventType: string; orgTitle: string }>(
```

Comment:
Changing a message like this (adding properties) is risky, because there might be translations that don't include the property. I think we should give the message a new name.